huxley/www/js/css/SupportLink.less

```diff
@@ -0,0 +1,28 @@
+.link-wrapper {
+  background-color: #111;
+  position: fixed;
+  bottom: 0pt;
+  right: 0pt;
+  text-align: right;
+  font-weight: 600;
+  padding: 5px 15px 5px 10px;
+  background-color: rgba(17, 17, 17, 0.6);
+  min-width: 135px;
+  border-top-left-radius: 6px;
+  border-top: 1px solid #222;
+  font-size: 11px;
+
+  .grey-text {
+    color: #aaa;
+  }
+
+  .mailto-link {
+    color: #eee;
+    font-size: 12px;
+
+    &:hover, &:active {
+      color: #aaa;
+      text-decoration: none;
+    }
+  }
+}
```

Comment:
@SachitShroff if you're using Sublime, you can configure it so that it adds a new line automatically whenever you save. Shoot me a message if you need help getting that set up.